packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -118,12 +118,13 @@ export interface ApiKeyConfig {
   readonly description?: string;
 
   /**
-   * The time from creation time after which the API key expires, using RFC3339 representation.
+   * The time from creation time after which the API key expires.
    * It must be a minimum of 1 day and a maximum of 365 days from date of creation.
    * Rounded down to the nearest hour.
-   * @default - 7 days from creation time
+   *
+   * @default - 7 days rounded down to nearest hour
```

Comment:
Why did you make this round to the nearest hour?

Reply:
The documentation [here](https://docs.aws.amazon.com/appsync/latest/APIReference/API_ApiKey.html) defines the `expires` parameter as:

> The time after which the API key expires. The date is represented as seconds since the epoch, rounded down to the nearest hour.

Reply:
@BryanPan342 Hmm, doesn't that mean though that they are doing the rounding? what happens if we send a 'non-rounded' epoch?

Reply:
Huh never tested that to be honest.. it probably rounds it for you but will try.

Reply:
oh sweet it does :) thanks for the insight